### DISCUSSION TITLE
Cluster favorites on the map for each group

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#588] [Q&A] [answered]
We now have a common setting in "Map | Appearance" to cluster (favorite) icons on the map.

We could define it separately for each group of favorites and with a different zoom level.

So more important groups with less favorites have visible all their icons.
While other less important groups with more favorites can be clustered.

**Comment by Ciutnik** [comment, marked as answer]
It's a cool idea, but won't it create additional confusion for some users through additional configuration options? 

On the other hand, perhaps as it is hidden in the "Extended Menu", beginners will not even notice it 🙂

**Reply by r11d11** [reply]
I would be happy if this setting would come. Most of the time I don't even think about this setting Map | Appearance | Group icons until zoom. For me it would be easier or clearer if I could define it separately
